### PR TITLE
add series component to crosswords

### PR DESCRIPTION
**Pull request by NathanielBennett**
Adds series to the crosswords
crosswords-series

### CODE REVIEW

File: applications/app/views/crossword.scala.html [hunk 1]
```diff
@@ -52,8 +52,7 @@ <h1 itemprop="headline" class="content__headline js-score">@Html(crosswordPage.c
             </header>
 
             <div class="content__main tonal__main tonal__main--tone-news">
-                <div class="gs-container">
-                    <div class="js-content-main-column">
+                <div class="gs-container"> <div class="js-content-main-column">
```
Comment (rich-nguyen) [inline applications/app/views/crossword.scala.html]: why?

File: static/src/javascripts/es6/projects/common/modules/crosswords/series.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/*eslint-disable no-new*/
+
+import qwery from 'qwery';
+import config from 'common/utils/config';
+import proximityLoader from 'common/utils/proximity-loader';
+import Series from 'common/modules/onward/onward-content';
+
+function insertOrProximity(selector, insert) {
+    if (window.location.href) {
```
Comment (rich-nguyen) [inline static/src/javascripts/es6/projects/common/modules/crosswords/series.js]: why test href?

File: static/src/javascripts/es6/projects/common/modules/crosswords/series.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        insert();
+    } else {
+        const el = qwery(selector)[0];
+
+        if (el) {
+            proximityLoader(el, 1500, insert);
```
Comment (rich-nguyen) [inline static/src/javascripts/es6/projects/common/modules/crosswords/series.js]: `proximityLoader` has an add function contained in an object, is this line working for you?

File: static/src/javascripts/es6/projects/common/modules/crosswords/series.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        }
+    }
+}
+
+
+export default() => {
+
+    insertOrProximity('.js-onwards', function () {
```
Comment (rich-nguyen) [inline static/src/javascripts/es6/projects/common/modules/crosswords/series.js]: this looks like a copy paste, i don't think you need insertOrProximity at all here. Pop all the code into default.

**Comment by rich-nguyen** [comment]
cool :+1:

File: static/src/javascripts/es6/projects/common/modules/crosswords/series.js (hunk 2 above)
Comment (NathanielBennett) [inline static/src/javascripts/es6/projects/common/modules/crosswords/series.js]: @rich-nguyen 
I just duplicated the functionality used on other pages here: https://github.com/guardian/frontend/blob/cbfcd4adeda892b10b0ed970878ec4b67db96104/static/src/javascripts/bootstraps/trail.js#L43 
Maybe that's over abstraction and I don't need this

File: static/src/javascripts/es6/projects/common/modules/crosswords/series.js (hunk 3 above)
Comment (NathanielBennett) [inline static/src/javascripts/es6/projects/common/modules/crosswords/series.js]: It appears to be - and it should be .add. I dont get that and I'm going to change that

**Pull request merged**